course/grader/grader.py

```diff
@@ -25,9 +28,10 @@ def grade(self, submission):
             else:
                 incorrect_count += 1
         if correct_count - incorrect_count == len(submission.question.answer.split(',')):
```

Comment:
This algorithm has a few problems. For instance, if the correct answer is `a,b` and the user enters `a,a` manually, they'll receive 100%.

I recommend creating a test for this and use the following test cases. Maybe add more if you want.

Correct Answer | User Answer | Grade
--- | --- | ---
a,b | a,a | 0%
a,b | b,a | 100%
a,b | !@#| 0%
a,b|b|50%
a,b|a,c|0%
a,b,c|a,b,d|33%
a,b,c|a,b,a|33%
a,b,c|`empty string`|0%
a|`empty string`|0%
a,b,c|a|33%
a,b,c|c,a|66%
a,b,c|e,f,a|0%
a|a|100%
a|b|0%